.gitignore

```diff
@@ -143,3 +143,6 @@ dmypy.json
 *.h5ad
 *.h5md
 *.h5mu
+
+# Test cache
+cache
```

Comment:
@wxicu this is required because elsewise we're inclined to track the cache folder if we're lazy and `git add .`